sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -604,68 +247,14 @@ def get_sdk_name_and_version():
     return (BEAM_SDK_NAME, beam_version.__version__)
 
 
-def get_sdk_package_name():
-  """For internal use only; no backwards-compatibility guarantees.
-
-  Returns the PyPI package name to be staged to Google Cloud Dataflow."""
-  sdk_name, _ = get_sdk_name_and_version()
-  if sdk_name == GOOGLE_SDK_NAME:
-    return GOOGLE_PACKAGE_NAME
-  else:
-    return BEAM_PACKAGE_NAME
+class DataFlowStager(stager.Stager):
```

Comment:
<!--new_thread; commit:d2647e7c0c6001a89be4dc0a673e17d5ce0d951a; resolved:0-->
See above, I don't think we need this class.